strings/strings.c

```diff
@@ -20,7 +26,16 @@ int string_length(char s[])
 */
 char *reverse_string(char rv[], char s[])
 {
-
+    int length = string_length(s):
```

Comment:
You have some small syntax errors, such as `:` instead of `;` here. Running `make tests` and going through the errors would resolve a lot of these minor issues.
```suggestion
    int length = string_length(s);
```